src/features/Classes/ClassesFilters/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Col, Form } from '@edx/paragon';
+import { Select, Button } from 'react-paragon-topaz';
+
+import { fetchAllCourses, fetchAllClassesData } from 'features/Common/data';
+import { updateCurrentPage, updateFilters } from 'features/Classes/data/slice';
+import { getClasses } from 'features/Classes/data';
+
+import { RequestStatus, initialPage } from 'features/constants';
+
+const initialStateTable = {
+  limit: true,
+  page: initialPage,
+};
+
+const ClassesFilters = () => {
+  const dispatch = useDispatch();
+  const instructorUserName = useSelector((state) => state.main.username);
+  const courses = useSelector((state) => state.common.allCourses);
+  const classes = useSelector((state) => state.common.allClasses);
+
+  const coursesList = courses.data
+    .map((course) => ({ label: course.masterCourseName, value: course.masterCourseName }));
+  const classesList = useSelector((state) => state.common.allClasses.data)
```

Comment:
Is really necessary to consult the state twise? Why just
```suggestion
   const courses = useSelector((state) => state.common.allCourses.data);
   const classes = useSelector((state) => state.common.allClasses.data);
  
  const couseOptions= courses.map((course) => ({
     label: course.masterCourseName, 
     value: course.masterCourseName
     }));
     
  const classOptions = classes.map((classItem) => ({
     label: classItem.className,
     value: classItem.classId
     }));
```